### ISSUE TITLE
Combat rule is 6 seconds now.

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#4973]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
You kidney shot target with 5 points for 6 seconds and should be able to get out of combat at 0.5 second left on kidney shot and sap the target. Right now rogue cannot leave combat before kidney shot runs out.
**Expected behaviour**: Tell us what should happen instead
You kidney shot target for 6 seconds, at 0.5 seconds left on the kidney shot you are able to leave the combat and sap the target.
**Steps to reproduce the problem**:

1. Get someone without stun reduction.
2. Put him into kidney shot for 6 seconds, press "esc" as soon as kidney shot lends.
3. You will be able to sap the target when he is out of kidney shot.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Jiaku,
I'm having a hard time understanding what you are trying to say here. If it is "Combat rule is 6 seconds now" considered an issue for you, it was discussed here #2426.
